src/Build/Graph/GraphBuildRequestData.cs

```diff
@@ -7,6 +7,14 @@
 
 namespace Microsoft.Build.Graph
 {
+    public record GraphBuildOptions
```

Comment:
Because there will be more things in it in the future and since MSBuild APIs are set in stone until the end of time I'd rather start with a 1 member record than successive API changes that progressively add yet another public constructor with yet another parameter followed eventually by an actual struct. Basically avoiding the fiasco with the Project constructors which got eventually followed by ProjectOptions.